lang/cs/Org.Apache.REEF.Client/YARN/Parameters/EnableDebugLogging.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using Org.Apache.REEF.Tang.Annotations;
+
+namespace Org.Apache.REEF.Client.YARN.Parameters
+{
+    [NamedParameter(
+        "Boolean indicating whether debug logging should be enabled in Java part of runtime.",
+        defaultValue: "false")]
+    public sealed class EnableDebugLogging : Name<bool>
```

Comment:
Has my earlier comment on this been resolved? My hunch was that something like this already existed. If not, it might make sense to do this as a separate JIRA to make sure that if we enable `DEBUG` logging, we do it on all Evaluators. Further, we might want to set the log level instead of using a boolean for this.